### DISCUSSION TITLE
Viewer toolbar text is garbled, missing icons and missing locale.js

### DISCUSSION THREAD

**Discussion by roldeguz** [mozilla/pdf.js#18158] [Q&A]
Attach (recommended) or Link to PDF file here:
[Bananas 24 Sep 2013.pdf](https://github.com/mozilla/pdf.js/files/15424456/Bananas.24.Sep.2013.pdf)

Configuration:
- Web browser and its version: Chrome 124.0.6367.208
- Operating system and its version: Windows 10
- PDF.js version: v4.2.67
- Is a browser extension: No

Steps to reproduce the problem:
1. Pass a pdf path to viewer.html

What is the expected behavior? (add screenshot)
The toolbar should be rendered properly with all the icons displaying, text is not garbled.
In Firefox, the viewer toolbar was rendered properly.
![image](https://github.com/mozilla/pdf.js/assets/7116954/a73e0c4c-f8cd-4975-bd10-ff5602993b2f)

What went wrong? (add screenshot)
Using Chrome, if you will notice, the text is garbled in the left side of the toolbar, and there were missing icons in the right side.
![image](https://github.com/mozilla/pdf.js/assets/7116954/22801031-297f-4e14-a999-ef08f8560386)

I also noticed that /locale/locale.js is missing. I checked the locale folder, the only file there is locale.json.

![image](https://github.com/mozilla/pdf.js/assets/7116954/4da1d27e-443c-4e88-b5e2-def72c864e8a)

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by roldeguz** [comment]
Just an update:

I managed to get it working by clearing the browser cache. (How did I not think of it before posting, I apologize.) But the console still displays a 404 for locale/locale.js from display_utils:412.

Thanks a lot!

**Comment by roldeguz** [comment]
Another update:

Aside from the 404 for locale/locale.js (display_utils:412), Im getting this another 404 error for /locale/en-US/viewer.ftl, which I can confirm that the file exists in that location.

![image](https://github.com/mozilla/pdf.js/assets/7116954/0f1e9e3f-1ae2-460c-88ab-63a9ef41e7a9)

**Comment by calixteman** [comment]
It's probably just a matter of configuring correctly your web server. I don't really see how we can help you.
